float32 speed
bool wait
---
string message